PC/pylauncher.rc

```diff
@@ -2,6 +2,11 @@
 
 #include "python_ver_rc.h"
 
+#ifndef RT_MANIFEST
+// bpo-45220: Cannot reliably #include RT_MANIFEST from
```

Comment:
The issue was with `#include <winuser.h>`, no?

Reply:
Does it matter? We no longer need to include any C++ headers here (you're not meant to include winuser.h directly anyway).

Reply:
It's just that I don't understand what `#include RT_MANIFEST` means. I guess that you mean that we cannot add an include to get the RT_MANIFEST constant. It's just a remark about the comment. You can ignore it ;-)